cpp/daal/src/services/service_topo.cpp

```diff
@@ -1923,29 +1923,21 @@ unsigned _internal_daal_GetStatus()
 
 unsigned getL1CacheSize()
 {
-    _internal_daal_GetSysProcessorCoreCount();
```

Comment:
Why not to delete this function at all?
To be sure, that it's not called in other places as well